### DISCUSSION TITLE
modrinth?

### DISCUSSION THREAD

**Discussion by tonbix** [maruohon/minihud#310] [Ideas]
I think it would be cool if the mod appeared on modrinth. So, will it?

**Comment by hexaclue** [comment]
I heavily agree!!!

**Comment by TheCrossCarrier** [comment]
Dear, can you answer us please! <3

**Comment by maruohon** [comment]
All of my mods will be on Modrinth. I've just been lazy about uploading them there... because it takes probably at least around 25 hours or active work per mod (for the client mods) to find and sort all my old mod builds, and to figure out and write the proper changelogs while uploading them. So I've taken long breaks between mods and upload sessions...

Currently malilib and Item Scroller are done, and MiniHUD will be next. Hopefully within a week or two from now...

**Reply by TheCrossCarrier** [reply]
You are the best bro! My thanks <3
